### PR TITLE
Feedback button

### PR DESCRIPTION
**Pull request by nemanja-da**
Changed the position of the 'Thumbs up/down' button to be in the bottom right corner above the 'Feedback' button. Added SASS to remove both buttons for mobile (as we were getting a lot of gibberish feedback

CHANGELOG_START
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by nemanja-da** [comment]
Ah, great point @cocreature 😄 You should be able to see it now. 

The problem is that Mopinion (our 3rd party tool used for the feedback form) deploys all the forms we ever wanna use on the website so I had to create rules what forms to show when (e.g., don't wanna show the "Tutorial feedback" form on docs and vice-versa don't wanna show docs' feedback form on Katacoda scenarios). 

Added a rule to show the form if it's also coming from http://127.0.0.1:8000/

image

File: docs/theme/da_theme_skeleton/layout.html [hunk 1]
```diff
@@ -268,7 +260,13 @@
     {% endif %}
   {% endif %}
 
+  {%- if url_root != './' %}
+  <div id="inpageContent"></div>
```
Comment (hurryabit) [inline docs/theme/da_theme_skeleton/layout.html:264]: This looks strange. Why do we put an empty `div` here? Is some JS going to render into this?

Comment (nemanja-da) [reply docs/theme/da_theme_skeleton/layout.html:264]: No, it's not gonna render. Just copy/pasted the old code that is used to determine if the page is docs.dam.com or not (if it is then don't show the feedback form)

Comment (hurryabit) [reply docs/theme/da_theme_skeleton/layout.html:264]: What happened to the `<a onclick=...>Detailed Feedback</a>` link?

Comment (cocreature) [reply docs/theme/da_theme_skeleton/layout.html:264]: It’s redundant. Previously, we had the blue feedback button that you can see in the screenshot at the bottom right and the detailed feedback button at the top and they both do the same thing. Now that the thumbsup/thumbsdown arrows are directly above the button on the bottom right there is no point in having the separate link.

**Comment by nemanja-da** [comment]
Added `localhost:8000` to the rule as well 😄

File: docs/theme/da_theme_skeleton/layout.html (hunk 1 above)
Comment (hurryabit) [reply docs/theme/da_theme_skeleton/layout.html:264]: That makes sense. Thanks for the explanation.

**Pull request merged**